src/cmlibs/exporter/webgl.py

```diff
@@ -61,6 +76,20 @@ def export_view(self):
     def _view_filename(self, name):
         return f"{self._prefix}_{name}_view.json"
 
+    def _define_default_LOD_obj(self, index):
+        LOD_obj = {
+            "Preload": False,
+            "Levels": {
+                "medium": {
+                    "URL": f"{self._prefix}_medium_{index}.json"
+                },
+                "close": {
```

Comment:
Are `medium` and `close` specified in the standard for setting the level of detail or can they be any label?

Reply:
Alan Wu said: "Those are set and can't be any other label unless I modify them."